root/static/scripts/common/MB/Control/Menu.js

```diff
@@ -32,35 +29,31 @@ MB.Control.HeaderMenu = function () {
         return 'auto';
     }
 
-    $('.header ul.menu > li').bind('mouseenter.mb', function (event) {
-        if (self.timeout) {
-            clearTimeout(self.timeout);
-            $('.header ul.menu li ul').css('left', '-10000px');
-        }
-
-        $(this).children('ul').css('left', getLeft(this));
-    });
-
-    $('.header ul.menu li').bind('mouseleave.mb', function (event) {
-        var ul = $(this).children('ul');
-
-        self.timeout = setTimeout(function () {
-            ul.css('left', '-10000px');
-        }, self.timeout_msecs);
-    });
-
-    $('.header .menu-header').on('click touchend', function (event) {
+    $('.header .menu-header').on('click', function (event) {
         event.preventDefault();
         event.stopPropagation();
         var ul = $(this).siblings('ul');
         $('.header ul.menu li ul').not(ul).css('left', '-10000px');
-        ul.css('left', ul.css('left') !== '-10000px' ? '-10000px' : getLeft(this.parentNode));
-        if ($.browser.android) {
-            $(this).parent().siblings().removeClass('fake-active');
+        var isClosing = ul.css('left') !== '-10000px';
+        ul.css('left', isClosing ? '-10000px' : getLeft(this.parentNode));
+        $('.header .menu-header').parent().removeClass('fake-active');
+        if (!isClosing) {
             $(this).parent().toggleClass('fake-active');
         }
     });
 
+    $('body').on('click', function (event) {
```

Comment:
This doesn't work at all on my phone (iOS). Works if I add `touchstart`, but that means I can't close a menu by clicking on the active header again (it closes the menu, then opens it again from the click event). There's definitely ways to work around it but might be more effort than it's worth...